DuckDuckGo/SiteRatingPrivacyProtectionExtension.swift

```diff
@@ -61,11 +61,19 @@ extension SiteRating {
     }
 
     func majorNetworksBlockedText() -> String {
-        return String(format: UserText.privacyProtectionMajorTrackersBlocked, uniqueMajorTrackerNetworksBlocked)
+        if 1 == uniqueMajorTrackerNetworksBlocked {
```

Comment:
Can we use plural dictionaries to avoid writing additional code here? They also translate better https://developer.apple.com/library/content/documentation/MacOSX/Conceptual/BPInternational/LocalizingYourApp/LocalizingYourApp.html

Reply:
That's what I was referring to in the description.  I'll have a look, but it doesn't look like quick job. If it is, I'll do it now but otherwise will have to do the release tomorrow.

Reply:
I think it's worth doing. The reason we put strings in `Localizable.strings` is so we can translate easily later it would be a shame to add breaking changes now. Happy to pair on it too if that makes the process less painful.